build.gradle

```diff
@@ -79,26 +80,27 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter-security'
     compile 'org.springframework.boot:spring-boot-starter-actuator'
     compile 'org.springframework.boot:spring-boot-configuration-processor'
+    compile 'org.springframework.boot:spring-boot-starter-oauth2-client:'
 
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlinVersion}"
     compile "org.jetbrains.kotlin:kotlin-reflect:${kotlinVersion}"
 
-    compile 'org.thymeleaf:thymeleaf-spring4:3.0.0.RELEASE'
-    compile 'org.thymeleaf:thymeleaf:3.0.0.RELEASE'
-    compile 'org.thymeleaf.extras:thymeleaf-extras-springsecurity4:3.0.0.RELEASE'
+    compile 'org.apache.httpcomponents:httpclient:4.5.12'
```

Comment:
Versie is niet nodig: https://docs.spring.io/spring-boot/docs/2.2.x/reference/html/appendix-dependency-versions.html#appendix-dependency-versions